doc/source/user/configoptions.rst

```diff
@@ -928,8 +928,9 @@ Networking options
 
 -  .. config:: GDAL_HTTP_USERAGENT
 
-      When set this string will be used to set the ``User-Agent`` header in the http
+      This string will be used to set the ``User-Agent`` header in the HTTP
       request sent to the remote server.
+      Defaults to "GDAL/x.y.z" where x.y.z is the current version.
 
```

Comment:
I wonder the wording "current version". Should it rather be "where x.y.z is the GDAL version" or "GDAL build version"? I think that the current GDAL version is always the most recent one https://gdal.org/download.html#current-release